src/interfaces/ITransformHelper.sol

```diff
@@ -0,0 +1,27 @@
+//SPDX-License-Identifier: None
+pragma solidity ^0.8.0;
+
+interface ITransformHelper {
+    function transformToCollateral(uint amount, bytes calldata data) external returns (uint);
+
+    function transformToCollateralAndDeposit(
+        uint amount,
+        address user,
```

Comment:
```suggestion
```

Reply:
Fixed in [6c6a1ef](https://github.com/InverseFinance/FiRM/pull/52/commits/6c6a1ef81d4a44de0e8dfbe92d7f59cf7108c22c)